### DISCUSSION TITLE
output cmd on label

### DISCUSSION THREAD

**Discussion by ljoli** [Textualize/textual#2311] [General]
Hello,

I like your works and i want to use it for use the output of cmd on my Linux ( Almalinux ).

So, How to display for example `uname -a` on a Widget ( like Label )  ?

**Comment by TomJGooding** [comment]
You might want to look into the Python [os](https://docs.python.org/3/library/os.html) and [subprocess](https://docs.python.org/3/library/subprocess.html) modules to learn about running external commands and getting its output.

**Reply by ljoli** [reply]
Hi, 

Effectively, in Python we have the module os and we can use like that : 
```
import os
 
os.uname()
```
( source : https://www.linuxfordevices.com/tutorials/linux/execute-linux-commands-python)

So, my question is how we can display this output on Textual ?

**Reply by liancheng** [reply]
A `Label` is also a `Static`, which has an [`update` method](https://textual.textualize.io/widgets/static/#textual.widgets._static.Static.update) to display any renderable object, which can be a string.
